package.json

```diff
@@ -21,7 +21,7 @@
     "prettier": "prettier --write \"**/*.{js,jsx,ts,tsx}\"",
     "publish-releases": "lerna publish from-git --no-push",
     "publish-releases:alpha": "yarn publish-releases --dist-tag alpha",
-    "start": "yarn workspaces foreach run start",
+    "start": "yarn workspaces foreach --interlaced --parallel --verbose run start",
```

Comment:
- `--interlaced` prints output as it comes in from the test commands, instead of buffering and printing all at once
- `--parallel` runs the commands in parallel
- `--verbose` prefixes each line of output with the name of the package that generated it

Reply:
checking for understanding -- this doesn't need `--topological` because we're watching the tokens build in that `start` script?

Reply:
Not exactly. Or, well, that's sort of it.

(For posterity) `--topological` makes the scripts run in dependency order. So first eds-tokens would run, then eds-components (since it has a dependency on eds-tokens).

That's great for the build scripts, since they run for a finite amount of time and then exit.

But `start` doesn't exit (until you ctrl-c it). So the eds-tokens start script would never exit, so the eds-components one never starts.

Hope that sort of made sense.